challenge.js

```diff
@@ -7,8 +7,8 @@
  *
  *	Answer below.
  */
-
-// ANSWER
+// ANSWER: In sublime, or another text reader, you'd add the appropriate tag, for example, a new heading tag could be
+// <h2>this element</h2>.  You would have to save the html document.  Or if you did have the page open in a browser, you could go to the js console and run something like document.addElement ....
```

Comment:
The first part of your answer would require reloading the page, and also being in control of the page source on the web server (suppose we wanted to add an element to the DOM in a google search tab). In the second part of your answer, `document.addElement` is not a method that exists. 1/2